docs/security.md

```diff
@@ -36,24 +36,46 @@ to the Spark Dispatcher instance.
 
 ### Binary Secrets
 
-When you need to store binary files into DC/OS secrets store, for example a Kerberos keytab file, your file needs to be base64-encoded as specified in RFC 4648.
+You can store binary files, like a Kerberos keytab, in the DC/OS secrets store. In DC/OS 1.11+ you can create
+secrets from binary files directly, while in DC/OS 1.10 or lower, files must be base64-encoded as specified in
+RFC 4648 prior to being stored as secrets.
+
+#### DC/OS 1.11+
+
+To create a secret called `mysecret` with the binary contents of `kerb5.keytab` run:
+
+```bash
+$ dcos security secrets create --file kerb5.keytab mysecret
+```
+
+#### DC/OS 1.10 or lower
+
+To create a secret called `mysecret` with the binary contents of `kerb5.keytab`, first encode it using the
+`base64` command line utility. The following example uses BSD `base64` (default on macOS).
 
-You can use standard `base64` command line utility. Take a look at the following example that is using BSD `base64` command.
 ```bash
 $ base64 -i krb5.keytab -o kerb5.keytab.base64-encoded
 ```
 
-`base64` command line utility in Linux inserts line-feeds in the encoded data by default. Disable line-wrapping via  `-w 0` argument.  Here is a sample base64 command in Linux.
+Alternatively, GNU `base64` (the default on Linux) inserts line-feeds in the encoded data by default.
+Disable line-wrapping with the `-w 0` argument.
+
 ```bash
 $ base64 -w 0 -i krb5.keytab > kerb5.keytab.base64-encoded
 ```
 
-Give the secret basename prefixed with `__dcos_base64__`. For example, `some/path/__dcos_base64__mysecret` and `__dcos_base64__mysecret` will be base64-decoded automatically.
+Now that the file is encoded it can be stored as a secret.
 
 ```bash
 $ dcos security secrets  create -f kerb5.keytab.base64-encoded  some/path/__dcos_base64__mysecret
 ```
-When you reference the `__dcos_base64__mysecret` secret in your service, the content of the secret will be first base64-decoded, and then copied and made available to your Spark application. Refer to a binary secret only as a file such that it will be automatically decoded and made available as a temporary in-memory file mounted within your container (file-based secrets).
+
+**Note:** The secret name **must** be prefixed with `__dcos_base64__`.
+
+When the `some/path/__dcos_base64__mysecret` secret is referenced in your `dcos spark run` command, its base64-decoded
```

Comment:
Question: Is it referenced in the command or in some service specification (as is used here: https://github.com/mesosphere/dcos-commons/blob/6da53edd1ff5392986bef5096a8c7a96470c64eb/docs/pages/_includes/services/overview.md#binary-secrets)? I'm ok leaving it as is though.

Reply:
In Spark, it's referenced in the Spark "run" command.